ui/raidboss/data/05-shb/trial/emerald_weapon-ex.js

```diff
@@ -122,36 +122,65 @@ export default {
+      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '5585', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'EmeraldEx Secundus Terminus est Corner',
+      netRegex: NetRegexes.startsUsing({ source: 'Bitblade', id: '55C9', capture: false }),
+      alarmText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go to intercards',
+          cn: '去场边四角',
+        },
+      },
+    },
+    {
       id: 'EmeraldEx Secundus Terminus est',
-      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55C8', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '55C8', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '55C8', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '55C8', capture: false }),
+      netRegex: NetRegexes.startsUsing({ source: 'Bitblade', id: '55CA', capture: false }),
       alarmText: (data, _, output) => output.text(),
       outputStrings: {
         text: {
-          en: 'X to cards, + to intercards',
-          cn: '靠边放剑(十字四角)',
-          ko: 'X는 동서남북, +는 대각위치로',
+          en: 'Go to cards',
+          cn: '去场边中间',
         },
       },
     },
     {
-      id: 'EmeraldEx Legio Phantasmatis',
-      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '55B4', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '55B4', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '55B4', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '55B4', capture: false }),
+      id: 'EmeraldEx Expire',
+      netRegex: NetRegexes.startsUsing({ source: 'The Emerald Weapon', id: '282b', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Smaragd-Waffe', id: '282b', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Émeraude', id: '282b', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'エメラルドウェポン', id: '282b', capture: false }),
+      response: Responses.getOut(),
+    },
+    {
+      id: 'EmeraldEx Mechanized Maneuver',
+      netRegex: NetRegexes.startsUsing({ source: `Black Wolf's Image`, id: '55BA', capture: false }),
       infoText: (data, _, output) => output.text(),
       outputStrings: {
         text: {
           en: 'Numbered Divebombs',
-          cn: '注意士兵顺序，结束返回中央',
+          cn: '观察飞机数字',
           ko: '엑사플레어 준비',
         },
       },
     },
+    {
+      id: 'EmeraldEx Full Rank',
+      netRegex: NetRegexes.startsUsing({ source: `Black Wolf's Image`, id: '55C0', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '观察士兵队列',
```

Comment:
@panicstevenson Do you have a suggestion of what this mechanic should say in English?

I think in general, all triggers need to have English because there's a lot of assumptions in the code that `en` always appears as a key.  I should probably write some tests that require this to make this more obvious.

Reply:
I see, that makes sense, I'll make sure to have English in the future. For this one, I'm not sure, what about ''Watch rank-and-file soldiers'

its a mech about finding the open spot for 3 lines of rank-and-file soldiers

Reply:
Yea, it's basically `watch rank-and-file soldiers`, I'd agree. I don't know currently if there's a way to call it out in the future like `start empty -> left -> right` or `start empty -> left -> left` etc in the future.

Reply:
sounds good,ill get those updated. not sure about the detection tho.. Is it based on position value?